src/Symfony/Component/Validator/Constraints/Unique.php

```diff
@@ -18,6 +18,8 @@
  * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
  *
  * @author Yevgeniy Zholkevskiy <zhenya.zholkevskiy@gmail.com>
+ * @author Marc Morera Merino <yuhu@mmoreram.com>
+ * @author Marc Morales Valldepérez <marcmorales83@gmail.com>
```

Comment:
Why these changes? And if, shouldn't they be done on the `4.3` branch?

Reply:
was from the original PR

Reply:
I think it needs to be revert because it was the contraints has been modfied in the original PR WDYT ? @xabbuh